addons/account/models/company.py

```diff
@@ -469,6 +469,9 @@ def _check_hash_integrity(self):
         """Checks that all posted moves have still the same data as when they were posted
         and raises an error with the result.
         """
+        if not self.env.user.has_group('base.group_system'):
```

Comment:
```suggestion
        if not self.env.user.has_group('base.group_account_user'):
```
`group_system` might be a little too restrictive, and would render the feature almost impossible to use.
I think using the same group as the one used in the view to enable the feature would make more sense.

Reply:
I have use the  same group of the menu to access of this report
https://github.com/odoo/odoo/blob/14.0/addons/account/views/account_menuitem.xml#L54

Reply:
I was not referring to the entire configuration menu item (which is more generic and less relevant in this case), but rather to the div where the button that triggers `_check_hash_integrity()` actually appears.
Anyways, please change the group to `group_account_user`.